Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -275,7 +300,7 @@ public AuthorizationClient(HttpClient client)
 
         #endregion
 
-        private void CheckIfNull(string value, string name)
+        private void CheckIfStringNullOrEmpty(string value, string name)

```

Comment:
Should an exception be thrown below?

Reply:
there is an AuthorizationException being thrown.